zerver/openapi/zulip.yaml

```diff
@@ -1581,7 +1581,7 @@ paths:
                                     **Changes**: Deprecated in Zulip 6.0 (feature level 148);
                                     starting with that feature level, `away` is a legacy way to
                                     access the user's `presence_enabled` setting, with
-                                    `away = !presence_enabled`. To be removed in a future release.
+                                    `away == !presence_enabled`. To be removed in a future release.
```

Comment:
I'm not sure I understand this one -- I think of this as our defining `away = !presence_enabled`, not doing a comparison.

Reply:
See this thread above: https://github.com/zulip/zulip/pull/25598#discussion_r1193866844